src/CBLUserAgent.hh

```diff
@@ -33,6 +33,22 @@ extern "C" NTSYSAPI NTSTATUS NTAPI RtlGetVersion(
 );
 #endif
 
+#if __APPLE__
+#include <sys/utsname.h>
+#if TARGET_OS_IPHONE == 1
```

Comment:
TARGET_OS_IPHONE also includes simulator. This can be changed to `TARGET_OS_IPHONE && !TARGET_OS_SIMULATOR` to the build error as `getDeviceModel` is not used.